src/web/components/ApiKeyManagement/KeyItem.scss

```diff
@@ -0,0 +1,6 @@
+.warning-button {
+  color: var(--theme-warning);
+  border-radius: 5px;
```

Comment:
Not sure if we need this `border-radius`?

Reply:
Yep thanks!